rider/src/main/kotlin/net/cakebuild/settings/CakeSettings.kt

```diff
@@ -19,14 +20,40 @@ class CakeSettings : PersistentStateComponent<CakeSettings> {
         }
     }
 
+    private val log = Logger.getInstance(CakeSettings::class.java)
+
     var cakeFileExtension = "cake"
     var cakeTaskParsingRegex = "Task\\s*?\\(\\s*?\"(.*?)\"\\s*?\\)"
     var cakeVerbosity = "normal"
-    var cakeRunner = "dotnet-cake"
-    var cakeRunnerOverrides = mapOf(Pair("^.*windows.*$", "dotnet-cake.exe"))
+    var cakeRunner = "~/.dotnet/tools/dotnet-cake"
+    var cakeRunnerOverrides = mapOf(Pair("^.*windows.*$", "\${USERPROFILE}\\.dotnet\\tools\\dotnet-cake.exe"))
```

Comment:
I'm wondering if it would make sense to support the standard syntax for environment variables e.g. `%VARIABLENAME%` on Windows and `$VARIABLENAME` on OSX/Linux, instead of a custom one `${VARIABLENAME}`.

What do you think?

Reply:
@augustoproiete I thought a lot about this and in the end decided that parsing `${FOO}` via Regex was the easiest thing to do. 

`$FOO` is only short for `${FOO}`. If I were to enable `$FOO` I'd have to decide where to stop the parsing. How would the equivalent to `${FOO}bar` look like? 

For `%FOO%` and or it's short form `%FOO` it's even harder. How to parse `%FOO%BAR%`?

Maybe we could - in some later version - add a simple "macro" editor where you can select some environment variables from a pre-selected list (or simply show all of them) and on click enter the correct `${SYNTAX}`.

Reply:
Yes, I get what you're saying. I was hoping you didn't have to parse anything, and that Kotlin would give you the equivalent of `System.Environment.ExpandEnvironmentVariables` (which does the parsing for you).

Reply:
I'd forgotten all about `ExpandEnvironmentVariables` but no - no such thing exists in OOTB java/kotlin.

Reply:
The code for parsing `%VARIABLENAME%` syntax is [here](https://github.com/dotnet/corert/blob/24f1edc938af9954eac9a3385958787ca64956d5/src/System.Private.CoreLib/shared/System/Environment.Unix.cs#L23), if you think it's worth converting to Kotlin to use this syntax.

Otherwise :shipit: as-is. Not an issue.